style/javascript.md

```diff
@@ -552,3 +553,53 @@ Yes:
 ```js
 $(".some-class span").hide();
 ```
+
+#### Don't use Underscore
+
+We use ES6/7 which includes many of the features of Underscore.js! Using Underscore should be avoided in favor of these native language features.
+
+There are a couple of methods that are sufficiently complicated and don't have a direct equivalent so instead we have a [custom-built](https://lodash.com/custom-builds) copy of [lodash](https://lodash.com/) containing only those specific methods. You can find this file at: `third_party/javascript-khansrc/lodash/lodash.js` along with instructions on how to build it and exactly what methods are included.
+
+What follows is a method-by-method set of equivalents for what Underscore provides and what you could be using in ES6/7 instead:
+
+Method | Use...                                | ...instead of
+--------- | ------------------------------------- | ----------------------
+bind | `fn.bind(someObj, args)` | `_.bind(fn, someObj, args)`
+bind | `() => { ... }` | `_.bind(function() { ... }, this)`
+bindAll | `obj.method = obj.method.bind(someObj);` <sup>[1](#u1)</sup> | `_.bindAll(someObj, "method")`
+clone | No alternative at the moment! |
+debounce | Our custom lodash build. <sup>[2](#u2)</sup> |
+defer | `setTimeout(fn, 0);` | `_.defer(fn);`
+delay | `setTimeout(fn, 2000);` | `_.delay(fn, 2000);`
+each | `array.forEach(fn)` | `_.each(array, fn)`
+each | `for (const val of array) {}` | `_.each(array, fn)`
+each | `for (const [key, val] of Object.entries(obj)) {}` | `_.each(obj, fn)`
+extend | `{...options, prop: 1}` | `_.extend({}, options, {prop: 1})`
+extend | `{...defaultOptions, ...options}` | `_.extend(defaultOptions, options || {})`
+extend | `Object.assign(json, this.model.toJSON())` | `_.extend(json, this.model.toJSON())`
```

Comment:
Here I would definitely say something like `extend [append]` vs `extend [assign]`, or some other way of making clear what the distinction between the two cases is.  Also, is it worth trying to combine the first two cases?